salt/modules/mysql.py

```diff
@@ -1110,7 +1110,15 @@ def alter_db(name, character_set=None, collate=None, **connection_args):
         collate or existing.get("collate"),
     )
     args = {}
-    _execute(cur, qry, args)
+    try:
+        if _execute(cur, qry, args):
+            log.info("DB '%s' altered", name)
+            return True
+    except MySQLdb.OperationalError as exc:
+        err = "MySQL Error {}: {}".format(*exc.args)
+        __context__["mysql.error"] = err
```

Comment:
Why are we keeping the error in `__context__`? Are we using it somewhere?

Reply:
That error in `__context__` is used in the state modules.

Reply:
:+1: